app-misc/qlcplus/qlcplus-4.12.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils udev virtualx xdg
+
+DESCRIPTION="A software to control DMX or analog lighting systems"
+HOMEPAGE="https://www.qlcplus.org/"
+SRC_URI="https://github.com/mcallegari/${PN}/archive/QLC+_${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/qlcplus-QLC-_${PV}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	|| ( dev-embedded/libftdi:1 dev-embedded/libftdi:0 )
```

Comment:
How is this library used? If the program links to it, then `||` is definitely wrong as you can't swap them without rebuilding it. Also, what happens if you have both slots installed?

Reply:
The build system automatically detects the version of libftdi installed. In case only SLOT 0 is installed, it is used. If libftdi:1 or both SLOTS are installed, libftdi-1.* is used.
You are right that they cannot be swapped without rebuilding but the software works when linked against either of them.
Could you propose a better solution to handle this in the ebuild? Thanks

Reply:
`dev-embedded/libftdi:=`. This will select the newest installed slot, and force rebuild when newer slot is installed.

Reply:
Done. However, this approach might lead to problems when a new SLOT of dev-embedded/libftdi is introduced and downstream packages are not yet compatible with it.
For the moment, it should be fine :)